tests/api/api.spec.ts

```diff
@@ -532,7 +532,74 @@ suite("@dataform/api", () => {
           tasks: [
             {
               type: "statement",
-              statement: "create or replace table `deeb.schema.name`  as select 1 as test"
+              statement: "create or replace table `deeb.schema.name` as select 1 as test"
+            }
+          ]
+        }
+      ];
+      const executionGraph = new Builder(testGraph, {}, dataform.WarehouseState.create({})).build();
+      expect(asPlainObject(executionGraph.actions)).deep.equals(
+        asPlainObject(expectedExecutionActions)
+      );
+    });
+
+    test("bigquery_clusterby", () => {
```

Comment:
We don't actually have an end to end test here checking that when we set cluster by on a table, it comes out in the final graph. This only checks from compilation onwards, and your example project only has a failure mode.

Reply:
Ah, have added test to core tests for both `partitionBy` and `clusterBy`